addons/marketing_campaign/models/marketing_campaign.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp.addons.decimal_precision import decimal_precision as dp
+from openerp import _, api, fields, models
+from openerp.exceptions import UserError
```

Comment:
reorganize import. first will be: 
from openerp import _, api, fields, models and then addons related import.